src/ansys/fluent/core/solver/flobject.py

```diff
@@ -159,6 +170,20 @@ def path(self) -> str:
             return self.obj_name
         return ppath + "/" + self.obj_name
 
+    @property
+    def python_path(self) -> str:
+        """Path of the object.
+
+        Constructed in python syntax from 'python_path' and the parents python
+        path.
+        """
+        if self._parent is None:
+            return "<session>"
+        ppath = self._parent.python_path
+        if not ppath:
+            return self.python_name
+        return ppath + self.python_name
```

Comment:
```suggestion
        return ppath + "." + self.python_name
```

Reply:
I did undo this change as I was getting a "." before the bracket:
![image](https://github.com/ansys/pyfluent/assets/143635850/93451d1e-c335-471f-95bf-0a5a758746e5)